src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/JSObject.References.cs

```diff
@@ -110,6 +85,11 @@ internal void DisposeImpl(bool skipJsCleanup = false)
             if (!_isDisposed)
```

Comment:
Looks like `_isDisposed` is never set to `true` on `FEATURE_WASM_THREADS`. Also `JSHandle` is not set to `IntPtr.Zero`.

Reply:
`_isDisposed` is set in the `JSProxyContext.ReleaseCSOwnedObject` under `lock`. I will add the `JSHandle` too. Thanks.